src/services/version_service/implementations/apiv1_version_service.ts

```diff
@@ -34,7 +34,8 @@ export class Apiv1VersionService extends VersionService {
 
   async fetchVersion(): Promise<Version> {
     try {
-      const { data } = await this.axios.get(`${this.baseUrl}/api/v1/version`);
+      const { data } = await this.axios.get(`${this.baseUrl}api/version/manifest`);
```

Comment:
Let's add typing to this response. I looked at the code, and it looks like when it was originally put in about 5 months ago, it was just typed with `object`. Now that we're switching let's add typing to the version object so we can treat it the same as the rest of the objects in the application.